crates/rlp/src/decode.rs

```diff
@@ -176,17 +176,17 @@ mod std_impl {
 #[inline]
 pub(crate) fn static_left_pad<const N: usize>(data: &[u8]) -> Result<[u8; N]> {
     if data.len() > N {
-        return Err(Error::Overflow)
+        return Err(Error::Overflow);
```

Comment:
hmm this is rustfmt I think?

Reply:
Yes, please run it with nightly: `cargo +nightly fmt`